src/test/java/StepDefinition/api/DashboardAPITestSteps.java

```diff
@@ -0,0 +1,85 @@
+package StepDefinition.api;
+
+import api.requests.*;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import utilities.ReadPropertyFile;
+import utilities.SharedData;
+
+import java.io.IOException;
+
+public class DashboardAPITestSteps {
+
+    SharedData sharedData = new SharedData();
+    ReadPropertyFile fr = new ReadPropertyFile();
+
+    @When("User creates the dashboard with  given dashboard name {string} and description {string} under the Project {string}")
+    public void userCreatesDashboardUsingAPIPost(String dashName,String  dashDescription, String projectName) throws IOException {
+        sharedData.setCreateDashboardRequest(CreateDashboardRequest.builder()
+                   .name(dashName)
+                   .description(dashDescription).build());
+        sharedData.setResponse(sharedData.getServiceUtil().postResponseFromService(
+                sharedData.getCreateDashboardRequest(), fr.getPropertyValue("PostDashboard.Url"), projectName,sharedData.getObjectMapper(),fr.getPropertyValue("AuthorizationToken")));
+    }
+
+    @Then("User receives {string} status code")
+    public void validateTheResponseStatusCode(String statusCode)
+    {
+        Assert.assertEquals(statusCode,String.valueOf(sharedData.getResponse().statusCode()));
+    }
+    @And("User validates the error code {string} and message {string}")
```

Comment:
New line is missed between methods.

Reply:
Please use the code formation with Ctr+Alt+L, to format the code as you have somewhere the Curly brackets in the different positions from method to method.